acceptance-test/user-pathways/happy-path/happy-path.test.js

```diff
@@ -50,7 +51,14 @@ describe('Critical user path(s)', () => {
 
     it('Happy path - Adult', async() => {
         try {
+            // start
             await clickContinueButton(1);
+
+            // who-do-you-work-for
+            await page.waitForSelector(ORGANISATION_INPUT);
+            await page.$eval(ORGANISATION_INPUT, (element) => {
+                element.value = 'Barnardos';
```

Comment:
let's talk about the value as being first letter uppercase and why it can't be all lowercase